tests/e2e/specs/auth-client-configuration.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeEach( async() => {
```

Comment:
Would be a bit more efficient to only activate the plugin once in `beforeAll`, especially because `activatePlugin ` is a no-op anyway when it's already active.

Also, it's best practice to tear down these things in `afterAll` / `afterEach`

Reply:
Right, beforeAll and afterAll used now.